### PR TITLE
manual: multiple http/https fixes

### PR DESCRIPTION
**Pull request by neteler**
- selected URL updates: fix missing full URLs to core manual pages
- multiple http/https fixes (some need to stay http)
- minor HTML cosmetics

### CODE REVIEW

File: src/raster/r.sun.daily/r.sun.daily.html [hunk 1]
```diff
@@ -43,5 +43,5 @@ <h2>SEE ALSO</h2>
 
 <h2>AUTHORS</h2>
 
-Vaclav Petras, <a href="http://geospatial.ncsu.edu/osgeorel/">NCSU OSGeoREL</a>,<br>
-Anna Petrasova, <a href="http://geospatial.ncsu.edu/osgeorel/">NCSU OSGeoREL</a>
+Vaclav Petras, <a href="https://geospatial.ncsu.eduosgeorel/">NCSU OSGeoREL</a>,<br>
+Anna Petrasova, <a href="https://geospatial.ncsu.eduosgeorel/">NCSU OSGeoREL</a>
```
Comment (wenzeslaus) [inline src/raster/r.sun.daily/r.sun.daily.html]: Wrong URL

Comment (wenzeslaus) [reply src/raster/r.sun.daily/r.sun.daily.html]: (missing slash)

Comment (wenzeslaus) [reply src/raster/r.sun.daily/r.sun.daily.html]: While at it, you can change it to: https://geospatial.ncsu.edu/geoforall/

File: src/raster/r.hydro.flatten/r.hydro.flatten.html [hunk 2]
```diff
@@ -79,4 +79,4 @@ <h2>SEE ALSO</h2>
 
 <h2>AUTHOR</h2>
 
-Anna Petrasova, <a href="http://geospatial.ncsu.edu/geoforall/">NCSU GeoForAll Lab</a>
+Anna Petrasova, <a href="https://geospatial.ncsu.edugeoforall/">NCSU GeoForAll Lab</a>
```
Comment (wenzeslaus) [inline src/raster/r.hydro.flatten/r.hydro.flatten.html]: Both "osgeorel" and "geoforall" variations are affected by the missing slash.

File: src/raster/r.sun.daily/r.sun.daily.html (hunk 1 above)
Comment (neteler) [reply src/raster/r.sun.daily/r.sun.daily.html]: Done in ac1dbb7380334ced839bfc6e5fbb4e9d86370b8d

File: src/raster/r.hydro.flatten/r.hydro.flatten.html (hunk 2 above)
Comment (neteler) [reply src/raster/r.hydro.flatten/r.hydro.flatten.html]: Ouch. Fixed in 4a4fcd044318e0211cace904208dd4f89dc870c1

**Comment by neteler** [comment]
Taking the silence as consent, moving on and merging.

**Pull request merged**